.github/workflows/ci.yml

```diff
@@ -16,7 +16,7 @@ jobs:
 
     - uses: actions/setup-python@v2
       with:
-        python-version: 3.7
+        python-version: "3.10"
```

Comment:
-1 on this

you do not ensure support for 3.7+

Reply:
find a way to satisfy all :)

Reply:
@beliaev-maksim I found the root cause. rolled back and set isort to 5.11.5. but pre-commit is still failing. There is no python27 on git.

    RuntimeError: failed to find interpreter for Builtin discover of python_spec='python2.7'

Reply:
@beliaev-maksim removed python2.7 from pre-commit. Please inspire me if there is a better solution.

Reply:
we might need to install it

but @boyang2022 can answer better

Reply:
@beliaev-maksim it is best to have both python 3.7 and 2.7 in pre-commit, but I have no idea how to setup two python environments. Can you help me on it?